Binary Search Tree/Solution 1/BinarySearchTree.playground/Sources/BinarySearchTree.swift

```diff
@@ -350,24 +322,23 @@ extension BinarySearchTree: CustomStringConvertible {
     }
     return s
   }
-}
-
-extension BinarySearchTree: CustomDebugStringConvertible {
-  public var debugDescription: String {
-    var s = "value: \(value)"
-    if let parent = parent {
-      s += ", parent: \(parent.value)"
-    }
-    if let left = left {
-      s += ", left = [" + left.debugDescription + "]"
-    }
-    if let right = right {
-      s += ", right = [" + right.debugDescription + "]"
-    }
-    return s
-  }
+   
+   public func toArray() -> [T] {
+      return map { $0 }
+   }
 
-  public func toArray() -> [T] {
-    return map { $0 }
-  }
 }
+
+//extension BinarySearchTree: CustomDebugStringConvertible {
```

Comment:
That was a mistake. I didn't intend to check it in.  However, I do feel the description is much more readable than the debugDescription.  It makes the playground much easier and fun to play with. So I still would like to recommend removing CustomDebugStringConvertible().